Makefile

```diff
@@ -0,0 +1,14 @@
+TMPDIR := $(shell mktemp -d)
+OUTDIR := $(shell pwd)
+
+_default:
+	@mkdir -p $(TMPDIR)/usr/share/anaconda/addons/
+	@cp -par org_fedora_hello_world $(TMPDIR)/usr/share/anaconda/addons/
+	
+	@cd $(TMPDIR)
+	@find . | cpio -c -o --quiet | gzip -9 > $(OUTDIR)/hello_world_addon_updates.img
```

Comment:
Please change the find part in `find .` to `find ./org_fedora_hello_world`.

Reply:
I pulled this find from[1].  I'll make it more precise, but I believe it does need to preserve the directory structure.

[1] https://rhinstaller.github.io/anaconda-addon-development-guide/sect-anaconda-addon-deploying-testing.html

Reply:
Ohh, I should have explained the problem. The point is that when you do `cp -par` to temp and then `find .` you will also add to the image other files which shouldn't be there (whole .git or IDE folders for example). 

You can use lsinitrd tool to inspect the image but inside of image should be *only* folder with addon which is `org_fedora_hello_world` in this case.

Maybe that find is not the correct solution but the `.git` folder really shouldn't be there.